python/setup.py

```diff
@@ -49,124 +50,9 @@
 
 inc_dir = finufftdir+"/include"
 src_dir = finufftdir+"/src"
+lib_dir = finufftdir+"/lib"
 finufft_lib = finufftdir+"/lib-static/finufft"
 
-class get_pybind_include(object):
-    """Helper class to determine the pybind11 include path
-
-    The purpose of this class is to postpone importing pybind11
-    until it is actually installed, so that the ``get_include()``
-    method can be invoked. """
-
-    def __init__(self, user=False):
-        self.user = user
-
-    def __str__(self):
-        import pybind11
-        return pybind11.get_include(self.user)
-
-# choose compile flags for finufftpy.cpp (links to finufft lib)...
-if sys.platform == "win32":
```

Comment:
I'm glad we're cleaning out the pybind11 stuff - but what is the plan for win32 and darwin specific settings like this - does ctypes work out of the box for them? Could you make such switches and FFTW library choices for win32 and darwin, please - thanks!

Reply:
A lot of it should reasonably go away since Python no longer has to compile any C++ sources. That being said, I agree that we need to test this to make sure nothing breaks.